sdk/formrecognizer/Azure.AI.FormRecognizer/tests/Infrastructure/FormRecognizerTestEnvironment.cs

```diff
@@ -51,6 +51,12 @@ public FormRecognizerTestEnvironment() : base("formrecognizer")
         /// <summary>The name of the PDF file which contains the multipage form to be used for tests.</summary>
         private const string MultipageFormFilename = "multipage_invoice_noblank.pdf";
 
+        /// <summary>The name of the PDF file which contains the multipage form with a blank page to be used for tests.</summary>
+        private const string MultipageBlankFormFilename = "multipage_invoice1.pdf";
+
+        /// <summary>The name of the PDF file which contains the single-page form with a blank page to be used for tests.</summary>
+        private const string BlankFormFilename = "blank.pdf";
```

Comment:
Mariana and I have discussed a plan to move these to another file and keep the `FormRecognizerTestEnvironment` organized, but that will be done in another PR.

Reply:
Thanks for proposing this Caio!!!